sitemedia/js/controllers/iiif_controller.js

```diff
@@ -144,6 +149,13 @@ export default class extends Controller {
                     viewer.viewport.zoomTo(1.1);
                 }
             });
+            // initialize desktop rotation buttons
+            this.rotateLeftTarget.addEventListener("click", () => {
+                viewer.viewport.setRotation(viewer.viewport.getRotation() - 90);
+            });
+            this.rotateRightTarget.addEventListener("click", () => {
+                viewer.viewport.setRotation(viewer.viewport.getRotation() + 90);
+            });
```

Comment:
These buttons could have been configured in the OSD initialization options (along with rotation increment), but since we're planning to customize further in when we implement the angle input, this seems fine.